client.go

```diff
@@ -1830,3 +1836,33 @@ func flags(f int) uint32 {
 	}
 	return out
 }
+
+// posixPerm converts Go permission bits to POSIX permission bits,
+// masking off file type bits.
+//
+// This differs from fromFileMode in that it preserves the POSIX versions of
+// setuid, setgid and sticky in m, because we've historically supported those
+// bits.
+func posixPerm(m os.FileMode) uint32 {
+	// POSIX mode bits, as listed at
+	// https://tools.ietf.org/html/draft-ietf-secsh-filexfer-13#section-7.6
```

Comment:
I would rather us reference the root authority for these values: `[IEEE.1003-1.1996]` rather than an indirect standard, which we specifically call out as not implementing.

Reply:
I've looked and cited the POSIX spec. Turns out this draft was subtly misrepresenting it.